.stylelintrc.js

```diff
@@ -1,8 +1,5 @@
 module.exports = {
-  extends: [
```

Comment:
Correction: The "extends" configuration must include both
 "stylelint-config-recommended-scss" and "@mate-academy/stylelint-config".
 When "extends" is defined as a single string (e.g., only "@mate-academy/stylelint-config"),
 Stylelint disregards the other configurations. To ensure all rules are applied,
 we keep "extends" as an array to combine the recommended SCSS rules
 with the "@mate-academy" configuration. This maximizes rule coverage and prevents unexpected behavior.